### ISSUE TITLE
Load models

### ISSUE THREAD

**Issue by kordjamshidi** [IllinoisCogComp/saul#411]
It seems in all of out tests the models are loaded from jar files. I was trying to read form my locally trained models but I am not sure how the path should be set, it is not working.

**Comment by danyaljj** [comment]
if [you don't specify the path](https://github.com/IllinoisCogComp/saul/blob/ce7a9eb367bf9854bc842b139e27bd93b6eb3d30/saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/Learnable.scala#L194) it will load the locally trained models.

**Comment by kordjamshidi** [comment]
it simply does not work for me, could you check any of these options I have or commented out in the t? I thought now we should use ClassifiersUtils  for loading models but at least for this example it does not workmen when I give '/models` as path or without specifying path.  @bhargav  https://github.com/IllinoisCogComp/saul/pull/409/commits/2cd6e5ec07600baf0b439db47e940cb690fe1b84#diff-0ff94b549b4b5b02d866c30adfe4bba6R22

**Comment by kordjamshidi** [comment]
@bhargav

**Comment by bhargav** [comment]
I looked at the SentimentApp.scala file and it seems that `sentimentClassifier` is never saved. The save function is called for SpamClassifier.

But we have a problem in the ClassifierUtils were `ClassifierUtils.LoadClassifier()` without model path does not default to the default model path.

**Comment by kordjamshidi** [comment]
Yes, right but the models for me where already in models directory, when I was running it. The problem is that `save()` stores them in `/saul/saul-examples/models` where we do not use save there are models in `/saul/models` we certainly need a fix here. It is very confusing. We lack tests that use load directly without reading from  model `.jar` files.
